From 41282f99aa1ce9fcd22346d38d9ac47f326ff5fa Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 Jan 2021 18:34:43 +0200
Subject: [PATCH] [SW-0] ppboot: fix camel case variable

No functional changes. Only fix a variable name from CameCase to a
regular underscore separated one.

Change-Id: I9668b1df9684d7eb37c7ea40df0464a67b63b66f
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/ppboot/ppboot.c b/ppboot/ppboot.c
index 2297620..08ac956 100644
--- a/ppboot/ppboot.c
+++ b/ppboot/ppboot.c
@@ -27,7 +27,7 @@
 }
 
 static bool ppboot_try_candidate_img(enum fw_post_reset_cmd image_pending,
-				     struct boot_desc *desc, bool isPri)
+				     struct boot_desc *desc, bool is_prime)
 {
 	bool is_valid = true;
 	u32 idx, bit;
@@ -55,7 +55,7 @@
 			preboot_puts("Checksum is not valid.\n");
 	}
 	if (!is_valid) {
-		if (isPri) {
+		if (is_prime) {
 			bit = CPU_BOOT_ERR0_PRI_IMG_VER_FAIL;
 			msg = "Primary";
 		} else {
